funcbench/.gitignore

```diff
@@ -0,0 +1 @@
+/funcbench
```

Comment:
can't this be in the global .gitignore file so we maintain only a single file?

Reply:
sure, will do it.